scripts/contrib/completion/bash/netctl

```diff
@@ -1,11 +1,19 @@
-#! /bin/bash
+#!/bin/bash
 
 GLOBAL_COMMANDS="\
+    version\
     group\
+    endpoint\
+    netprofile\
     network\
     tenant\
     policy\
+    external-contracts\
     global\
+    aci-gw\
+    bgp\
+    app-profile\
+    service\
     help"
```

Comment:
This is actually the current order of the command output from `netctl --help`

We need to reorder the `Commands` array in `netctl/commands.go`, too.